### PR TITLE
Add Masking plots

### PR DESCRIPTION
**Pull request by andersy005**
Towards #13

- NCL Version:

- Python Version:

sphx_glr_mask_1_002

- NCL Version:

- Python Version:
sphx_glr_mask_1_001

### CODE REVIEW

File: Plots/Masking/mask_1.py [hunk 1]
```diff
@@ -0,0 +1,187 @@
...
+import cartopy.crs as ccrs
+import cmaps
+import matplotlib as mpl
+import matplotlib.pyplot as plt
+import numpy as np
+import xarray as xr
+
+# Read in the netCDF file
+
+ds = (
```
Comment (clyne) [inline Plots/Masking/mask_1.py]: Could you please comment why the decode_times keyword is needed?

File: Plots/Masking/mask_1.py [hunk 2]
```diff
@@ -0,0 +1,187 @@
...
+
+
+# Use xarray's where function to mask out land and then ocean data
+land_only = ds.TS.where(ds.ORO == 1.0)
+ocean_only = ds.TS.where(ds.ORO == 0.0)
+
+
+# Define a few utility functions
+
+def truncate_colormap(cmap, minval=0.0, maxval=1.0, n=100):
```
Comment (clyne) [inline Plots/Masking/mask_1.py]: Could you please explain  a little more about what truncate_colormap is doing (and maybe why it is needed)

File: Plots/Masking/mask_1.py [hunk 3]
```diff
@@ -0,0 +1,187 @@
+"""
```
Comment (clyne) [inline Plots/Masking/mask_1.py]: The plots have a gap along 0 degrees longitude (wraparound). Can this be fixed? I think @jukent had a solution.

**Comment by andersy005** [comment]
New plots:

sphx_glr_mask_1_001 (1)
sphx_glr_mask_1_002 (1)

File: Plots/Masking/mask_1.py (hunk 3 above)
Comment (andersy005) [reply Plots/Masking/mask_1.py]: This is fixed

File: Plots/Masking/mask_1.py (hunk 2 above)
Comment (andersy005) [reply Plots/Masking/mask_1.py]: Added a note in the documentation string of the function

**Pull request merged**